src/desktop/components/view_in_room/index.jade

```diff
@@ -3,10 +3,10 @@
 img.view-in-room__artwork( class='js-view-in-room-artwork' )
 
 a.view-in-room__close( class='js-view-in-room-close' )
-  i.icon-close
+  | Close
```

Comment:
gave up trying to get that icon to work /shrug

Reply:
`&times;`?

Or if you want an adjustable CSS + that you can rotate: https://github.com/auspices/eos/blob/42978b9e30a8dc2131f2eff439acb1e0230cb5f6/src/Plus/Plus.tsx 😆

Reply:
Oh yah this is way better 👍